packages/shortest/src/commands/init/index.ts

```diff
@@ -0,0 +1,124 @@
+import { execSync } from "child_process";
+import { readFile, writeFile } from "node:fs/promises";
+import { join } from "path";
+import { fileURLToPath } from "url";
+import { detect, resolveCommand } from "package-manager-detector";
+import pc from "picocolors";
+import { CONFIG_FILENAME, ENV_LOCAL_FILENAME } from "../../constants";
+import { addToEnv } from "../../utils/add-to-env";
+import { addToGitignore } from "../../utils/add-to-gitignore";
+
+export default async function main() {
+  console.log(pc.blue("Setting up Shortest..."));
+
+  try {
+    const packageJson = await getPackageJson();
+    if (
+      packageJson?.dependencies?.["@antiwork/shortest"] ||
+      packageJson?.devDependencies?.["@antiwork/shortest"]
+    ) {
+      console.log(pc.green("✔ Package already installed"));
+      return;
+    } else {
+      console.log("Installing @antiwork/shortest...");
+      const installCmd = await getInstallCmd();
+      execSync(installCmd, { stdio: "inherit" });
+      console.log(pc.green("✔ Dependencies installed"));
+    }
+
+    const configPath = join(process.cwd(), CONFIG_FILENAME);
+    const exampleConfigPath = join(
+      fileURLToPath(new URL("../../src", import.meta.url)),
+      `${CONFIG_FILENAME}.example`,
+    );
+
+    const exampleConfig = await readFile(exampleConfigPath, "utf8");
+    await writeFile(configPath, exampleConfig, "utf8");
+    console.log(pc.green(`✔ ${CONFIG_FILENAME} created`));
+
+    const envResult = await addToEnv(process.cwd(), {
+      ANTHROPIC_API_KEY: {
+        value: "your_value_here",
+        comment: "Shortest variables",
+      },
+    });
+    if (envResult.error) {
+      console.error(
+        pc.red(`Failed to update ${ENV_LOCAL_FILENAME}`),
+        envResult.error,
+      );
+    } else if (envResult.added.length > 0) {
+      const added = envResult.added.join(", ");
+      const skipped = envResult.skipped.join(", ");
+      const detailsString = [
+        added ? `${added} added` : "",
+        skipped ? `${skipped} skipped` : "",
+      ]
+        .filter(Boolean)
+        .join(", ");
+      console.log(
+        pc.green(
+          `✔ ${ENV_LOCAL_FILENAME} ${envResult.wasCreated ? "created" : "updated"} (${detailsString})`,
+        ),
+      );
+    }
+
+    const resultGitignore = await addToGitignore(process.cwd(), [
+      ".env*.local",
```

Comment:
Not worth the effort trying to group with other `env*`-like values already present (e.g. `.env` from a fresh Next.js install).